proto/buf/alpha/registry/v1alpha1/repository_commit.proto

```diff
@@ -163,3 +167,20 @@ message DeleteRepositoryDraftCommitRequest {
 }
 
 message DeleteRepositoryDraftCommitResponse {}
+
+message ListBSRHeadHistoryRequest {
```

Comment:
Why do we need another endpoint for this? What does this mean? Can we have docs on this as well?

Reply:
This ended up being removed, but it was documented in the RPC definition in the service. In the future, I can add docs to the request and response messages as well.